compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/java/KSClassDeclarationJavaImpl.kt

```diff
@@ -78,6 +78,7 @@ class KSClassDeclarationJavaImpl private constructor(val psi: PsiClass) : KSClas
             it.unsubstitutedMemberScope.getDescriptorsFiltered(DescriptorKindFilter.FUNCTIONS)
                 .toList()
                 .filter { (it as FunctionDescriptor).visibility != Visibilities.INVISIBLE_FAKE }
+                .plus(it.constructors)
```

Comment:
Do getAllFunctions() of KSClassDeclarationImpl and KSClassDeclarationDescriptorImpl contain constructors? primary, synthesized and defined.

Reply:
Yes, see test case in allFunctions.kt